Orange/base.py

```diff
@@ -244,9 +245,17 @@ def __call__(self, data, ret=Value):
             prediction = self.predict(data)
         elif isinstance(data, (Table, Instance)):
             if isinstance(data, Instance):
-                data = Table(data.domain, [data])
-            if data.domain != self.domain:
-                data = data.transform(self.domain)
+                trans_data = Table(data.domain, [data])
+            elif data.domain != self.domain:
```

Comment:
The first elif should stay as if. The domain of a single instance was not yet converted.